torch/csrc/jit/tensorexpr/tensorexpr_init.cpp

```diff
@@ -210,7 +221,22 @@ void initTensorExprBindings(PyObject* module) {
       },
       py::return_value_policy::reference);
 
+  te.def(
+      "Reduce",
+      [](const std::string& func_name,
+         const std::vector<DimArg>& dim_args,
+         const Reducer& reducer,
+         const BufHandle& buffer,
+         const std::vector<DimArg>& reduce_args) {
+        return Reduce(func_name, dim_args, reducer, buffer, reduce_args);
+      },
+      py::return_value_policy::reference);
+
   py::class_<Stmt, std::unique_ptr<Stmt, py::nodelete>>(te, "Stmt")
+      .def(py::init([](const std::vector<Stmt*>& stmts) {
+        return std::unique_ptr<Stmt, py::nodelete>(
```

Comment:
Same question about unique_ptr. I don't mean we definitely should change it - I'm just trying to understand it better :)